src/services/mapCenterApi.ts

```diff
@@ -1,26 +1,10 @@
 import axiosClient from "@/config/client";
 import { handleAxiosError } from "@/util/error";
 
-// Function to Fetch Map Center
```

Comment:
Hi @DarinHajou - If you are removing this endpoint, how do you plan to fetch the map center for each user?  Perhaps you know something that I don't.  
Regardless, removing from the frontend would also mean cleaning up the backend.  Could you please clarify your approach?  Thanks!

Reply:
I accidentally removed the fetchMapCenter function from the frontend. This function is necessary to fetch the saved map center coordinates for users when they revisit the map, and it interacts with the getMapCenter controller on the backend.

I initially thought it was redundant because our current setup sends the coordinates directly to the backend when a user sets their map center. But after reviewing it again, I see that this function is still needed for returning users to view their saved map center.

I've corrected the issue by adding the fetchMapCenter function back. Apologies for the confusion, and thanks for pointing it out.